design/queue.md

```diff
@@ -0,0 +1,26 @@
+# Executor Queue
+
+## Context
+
+This is an executor plugin for Screwdriver that makes use of [Resque][node-resque-URL] to add a queueing mechanism.
+
+## Architecture Overview
+
+![](./diagrams/executor-queue-architecture.puml.png)
+
+## Design Decisions
+
+### Router
+
+The `executor-queue` will sit behind `executor-router` until it works as we expect, so as not to disturb people already using `executor-router`.
```

Comment:
Should there be a section talking about future plans, like combining the queue and the router and/or having multiple workers (with each worker responsible for a specific executor)?